air/src/lib.rs

```diff
@@ -21,6 +21,12 @@ pub use options::ProofOptions;
 pub use vm_core::{utils::ToElements, Felt, FieldElement, StarkField};
 pub use winter_air::{FieldExtension, HashFunction};
 
+// CONSTANTS
+// ================================================================================================
+
+/// The length of a cycle in the periodic columns.
+pub const PERIODIC_CYCLE_LEN: usize = 8;
```

Comment:
It might be useful to name this for the processor, since the hasher also uses periodic cycles, and it's possible that in the future we'll change the bitwise processor to have cycles of length 4. Of course at the moment that would mean duplicate constants, since they're currently the same length. What do you think?

Reply:
I think it's fine to move this to the processor level - but also, this is a very minor nit - so, I'm fine either way.

Reply:
I moved it but kept the name. I feel like it's descriptive enough, and matches with the naming of the periodic columns, so to me it makes sense. I think it will be clearer keeping them in the processor contexts though.